portfolio/src/main/webapp/script.js

```diff
@@ -13,20 +13,28 @@
 // limitations under the License.
 
 /**
- * Show/hide the description inside the element which has title as id.
- */
-function toggleContent(sectionId) {
-  if (document.getElementById(sectionId) == null) {
-      return;
+* Insert/remove the class in the element.
+*/
+function toggleClass(element, classAttribute) {
+  if (!element || typeof element.className !== "string" || !classAttribute || typeof classAttribute !== "string") {
+    return;
+  }
+  const classes = element.className.split(" ");
+  if (classes.indexOf(classAttribute) === -1) {
```

Comment:
There is also `.includes()` which returns a boolean.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

(I know it doesn't work in IE but usually we could use a polyfill or transpile step to make that work)

Reply:
Actually since you reuse the value of this below to remove the class, instead extract it into a constant and use in both places.